### DISCUSSION TITLE
driver.save_cookies doesnt work.

### DISCUSSION THREAD

**Discussion by Vrackko** [seleniumbase/SeleniumBase#2840] [Q&A] [answered]
Hi i need my script to save cookies from websites that would later be used. I am using Driver.

driver.delete_all_cookies() seems to work fine

driver.save_cookies(name="cookies.txt") give this error:
Traceback (most recent call last):
  File "c:\Users\HP\Desktop\No Com\run.py", line 88, in <module>
    driver.save_cookies(name="cookies.txt")
    ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\Desktop\No Com\.venv\Lib\site-packages\seleniumbase\undetected\__init__.py", line 326, in __getattribute__
    return super().__getattribute__(item)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Chrome' object has no attribute 'save_cookies'

Should i just use some of selenium driver methods to do cookies or i can use this one.

**Comment by mdmintz** [comment, marked as answer]
See https://github.com/seleniumbase/SeleniumBase/discussions/2705#discussioncomment-9176117
`sb` has all the cookie methods (not the `driver`).

```python
sb.save_cookies(name="cookies.txt")
sb.load_cookies(name="cookies.txt")
sb.delete_all_cookies()  # Duplicate: self.clear_all_cookies()
sb.delete_saved_cookies(name="cookies.txt")
sb.get_saved_cookies(name="cookies.txt")
sb.get_cookie(name)
sb.get_cookies()
sb.add_cookie(cookie_dict)
sb.add_cookies(cookies)
```

**Reply by Vrackko** [reply]
I am using driver for all of my script, is there no driver variant?

**Reply by mdmintz** [reply]
The `driver` has a few cookie methods that come with raw Selenium. (Not the same ones that SeleniumBase has.)
